security advisory message added to 8.18.1 release notes (#17642)
elastic/logstash: docs/static/releasenotes.asciidoc

[[logstash-8-18-1]]
=== Logstash 8.18.1 Release Notes

[IMPORTANT]
====

> The 8.18.1 release contains fixes for potential security vulnerabilities. See our https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for more details.

Reviewer: ```suggestion
The 8.17.6 release contains fixes for potential security vulnerabilities.
Check out the https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for details.
```

[[logstash-8-17-6]]
=== Logstash 8.17.6 Release Notes

[IMPORTANT]
====

> The 8.17.6 release contains fixes for potential security vulnerabilities. See our https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for more details.
> ====
> 
> No user-facing changes in Logstash.

Reviewer: ```suggestion
The 8.17.6 release contains fixes for potential security vulnerabilities.
Check out the https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for details.
```

[[logstash-8-17-6]]
=== Logstash 8.17.6 Release Notes

> [IMPORTANT]
> ====
> The 8.17.6 release contains fixes for potential security vulnerabilities.
> Check out the https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for details.

Reviewer: ```suggestion
[IMPORTANT]
====
The 8.17.6 release contains fixes for potential security vulnerabilities.
Check out the https://discuss.elastic.co/c/announcements/security-announcements/31[security advisory] for details.
====
```
